playground/dist/index.html

```diff
@@ -111,6 +111,7 @@ <h1 class="vqb__drop-zone__message">
     <!-- built files will be auto injected -->
     <script src="https://cdn.jsdelivr.net/npm/vue/dist/vue.js"></script>
     <script src="https://unpkg.com/vuex@3.1.1/dist/vuex.js"></script>
+    <script src="/mingo.min.js"></script>
```

Comment:
We could do the same as above, i.e. using a CDN to deliver mingo.
I think that would be better that to add this in the main package.json (we don't want our users to have to download mingo to use the weaverbird).
Another option would be to have a dedicated package.json for the playground.

That second option seems to me like the best idea, I see other dependencies that would be nice to exclude from today's deps (mongo, mongo-prebuilt, express,... everything related to the playground)